docs/source/add_new_adaptor.md

```diff
@@ -0,0 +1,243 @@
+
+The diagram below illustrates all the relevant steps of how adaptor is invoked, with additional details provided for each annotated step.
+```mermaid
+  sequenceDiagram
+  	autonumber
+    Strategy ->> Adaptor: query framework capability
+    Adaptor ->> Strategy: Parse the framework YAML and return capability
+    Strategy ->> Strategy: Build tuning space
+    loop Traverse tuning space
+    	Strategy->> Adaptor: generate next tuning cfg
+        Adaptor ->> Adaptor: calibrate and quantize model based on tuning config
+
+    end
+```
+1. **Strategy**: Drives the overall tuning process and utilizes `adaptor.query_fw_capability` to query the framework's capabilities.
+
+2. **Adaptor**: Parses the framework YAML, filters some corner cases, and constructs the framework capability. This includes the capabilities of each operator and other model-related information.
+
+3. **Strategy**: Constructs the tuning space based on the framework capability and initiates the tuning process.
+
+4. **Strategy**: Generates the tuning configurations for each operators of the model using the tuning space constructed in the previous step, specifying the desired tuning process.
+
+5. **Adaptor**: Invokes the specific kernels for the calibration and quantization based on the tuning configuration.
+
+
+## Add query_fw_capability to Adaptor
+
+To enable accuracy-aware tuning with a specific framework, we should define the [framework YAML](./framework_yaml.md) which unifies the configuration format for quantization and provides a description for the capabilities of the specific framework. 
+>**Note**: You should refer to [framework_yaml.md](./framework_yaml.md) to define the framework specific YAML.
+
+After the framework YAML enabled, The The abstract class `QueryBackendCapability` should be inherited and implemented. This class is used to load the YAML file and parse the quantization config from the specific framework version. You can refer to Tensorflow implementation [TensorflowQuery](../neural_compressor/adaptor/tensorflow.py#L628).
+
+With the inherited `QueryBackendCapability` class enabled. The Adaptor will initialize a `query_handler` and used in API `query_fw_capability`.
+
+Each framework adaptor should implement the `query_fw_capability` fucntion, this function will only be invoked once and will loop over the graph/model for the quantizable operators and collect each operator's opwise details and optypewise capability. You should return a standard dict of the input model's tuning capability. The format is like below:
+
+```python
+    def query_fw_capability(self, model):
+        # loop over the model and generate the capability of the model
```

Comment:
this code snippet is also not easy to understand, such as where the quantiable_op_details variable comes from? whether it's the whole code or just partial of code? should use `...` to present those skipped code?

Reply:
you didn't solve my concerns.

I think the doc is too complex and expose too much internals to user like the explanation on `quantizable_op_details`